docs/adr/0031-process-evolution.md

```diff
@@ -0,0 +1,171 @@
+# FxA Process Evolution
+
+- Status: proposed
+- Deciders: Wil Clouser, Ivo Plamenac, Jon Buckley, Dustin Lactin
+- Date: 2022-06-02
+
+## Context and Problem Statement
+
+We have a [documented process](https://mozilla.github.io/ecosystem-platform/reference/team-processes/development-process)
+today (see also the [release owner duties](https://mozilla.github.io/ecosystem-platform/reference/team-processes/release-process)),
+but it’s complex and can be error-prone. It was built before we have the tools
```

Comment:
Should "It was built" be "our release script"?

Reply:
The "it" here is our whole release process, including our release script.  I'll update the wording.